### ISSUE TITLE
Flash gateway using non solder option

### ISSUE THREAD

**Issue by aserno** [serrj-sv/lumi.gateway.mgl03#18]
I was trying to downgrade the gateway with this method. Got the pcb clip connected on the Board with the ttl to usb 3.3v jumper set(as was suggested) windows sees the usb module after a driver install(com4)

next step connect it with de tera term.
Script says connect it with poweroff. Then power on.. after that.. nothing. What should i expect to see after the gateway boots up? Script doesn’t change after the “power on” what do i miss?

**Comment by sinad07** [comment]
Неправильное подключение ,поменяйте местами rx - tx00:00, 2 февраля 2021 г., aserno <notifications@github.com>:
I was trying to downgrade the gateway with this method. Got the pcb clip connected on the Board with the ttl to usb 3.3v jumper set(as was suggested) windows sees the usb module after a driver install(com4)
next step connect it with de tera term.
Script says connect it with poweroff. Then power on.. after that.. nothing. What should i expect to see after the gateway boots up? Script doesn’t change after the “power on” what do i miss?

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by marrubio** [comment]
I have the same issue with soldering option. 

The usb to serial converter seems to work correctly once the drivers are installed, but there seems to be no communication, I have checked the welds with a tester and they seem to work without apparent problems, current firmware version v1.4.6_0043

**Comment by sandrwich** [comment]
I just solved my problem similar to this one. I have tried many times with different UART sticks with no success - always no output when trying to connect. 

Then I stepped at the post regarding serial connections on the sparkfun containing this line:
> So the RX from one device should go to the TX of the other, and vice-versa.

At the end I figured out that in fact you have to connect Rx on gateway to Tx on UART, not Rx to Rx. It may be not that obvious to someone who is making serial connection for the first time.

**Comment by serrj-sv** [comment]
its basically mentioned in  manual:

2. connect UART:
UART Tx <--> TP4 (Gateway Rx)
UART Rx <--> TP11 (Gateway Tx)
UART GND <--> TP8 (Gateway GND)

**Comment by sandrwich** [comment]
> its basically mentioned manual: 2. connect UART: UART Tx <--> TP4 (Gateway Rx) UART Rx <--> TP11 (Gateway Tx) UART =

Yes, I just noticed it and wanted to add a note in my first post. I probably assumed my Tx-Tx and Rx-Rx makes sense and proceeded to the next step. I think it is possible mistake to be made and is a probable cause of the issues described here.

**Comment by marrubio** [comment]
Thank you very much. That was de problem, the "TX" and "RX" cables were NOT crossed, I didn't read that part of the tutorial

**Comment by Tualua** [comment]
Hello!
![image](https://github.com/user-attachments/assets/12b7fb76-89f9-4e34-94bb-d0bbfc564d10)

My gateway throws garbage to UART. Is this a problem with the gateway itself or am I doing something wrong?
I am using FTDI TTL to USB cable and solderless clip
![1209295e792c162a6e9e9814433fcba](https://github.com/user-attachments/assets/7c590b9a-9589-4ff4-abd1-b5cddbfcd424)

bitrate is 115200, but I tried to use other speed with the same result
